hermes-schema/src/test/java/pl/allegro/tech/hermes/schema/schemarepo/SchemaRepoSchemaSourceClientTest.java

```diff
@@ -0,0 +1,240 @@
+package pl.allegro.tech.hermes.schema.schemarepo;
```

Comment:
why is it in java and not groovy? this is weird, as other tests are in groovy in this module